### DISCUSSION TITLE
Is it possible to forward messages from public Chanels where you have no admin rights?

### DISCUSSION THREAD

**Discussion by ionedji** [aahnik/tgcf#475] [Q&A] [answered]
Hi there. My goal is to forward messages from public channels to my own but the issue is that i`m not admin in those channels (cause they are not belong to me). 

Currently I`m able only to forward messages from my chanel#1 to chanel#2 (cause I added bot as admin to both of them). 

Is it possible to forward messages from public Chanels where you have no admin rights (not able to add bot as admin of the channel)?

**Comment by 25gabriel-cmyk** [comment, marked as answer]
Yes you can, use User Session for your Telegram login, NOT the Bot. Make sure your telegram account subscribed to source channel

**Reply by ionedji** [reply]
Thanks for the hint!

**Comment by MobCode100** [comment]
Yes you can use your user account to do this, but the downside is you can't see notifications for new messages as you are the one who sends it. For me, it is convenient to see notifications at my own backup channels/groups. Thus, tells me that tgcf is still working and my backup group/channel is in sync with the original one. The only way to solve this is to use another telegram account other than our own.

For someone who needs to forward messages from public channels, using bot/other user account, I've made a PR #478, this plugin allows you to change the sender/forwarder account. You will login both accounts at the same time which are the user that's subscribed to the public channel and also the bot/other user account that you want to use as the sender. I'm waiting for it to be merged, hopefully :pray:. Anyways, the feature is at my forked [repo](https://github.com/MobCode100/tgcf-sender-plugin/tree/latest) feel free to try it. Thank you! :smile:
